src/iotjs_binding.h

```diff
@@ -216,4 +219,37 @@ jerry_value_t iotjs_jhelper_eval(const char* name, size_t name_len,
 
 jerry_value_t vm_exec_stop_callback(void* user_p);
 
+/**
+ * Dynamic module defintions (.iotjs)
+ */
+
+#define IOTJS_MAKE_VERSION(major, minor, patch)               \
+  (((0x10 & 0xFF) << (3 * 8)) | ((major & 0xFF) << (2 * 8)) | \
+   ((minor & 0xFF) << (1 * 8)) | (patch & 0xFF))
```

Comment:
Patch? A major/minor should be enough. Well I think even a major is enough with uint32 range.